proposals/20200915-prow.md

```diff
@@ -0,0 +1,52 @@
+# Prow on AWS
+
+This document reflects the way we test Falco Security using Prow, and how it should be built, and maintained.
+
+## Terms & Definitions
+
+- [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) is an open source testing framework built on top on Kubernetes.
+
+## Motivation
+
+Today Falco pull requests use github automation for labels via Prow. The proposal is to fully setup prow testing infrastructure on top of AWS to utilize the functional testing aspect of prow. This will provide users, and maintainers a streamlined way to display test results, and control testing workflows.
+
+## Goals
+- To Setup test workflow for [Falco](https://github.com/falcosecurity/falco) and repositories needing testing on EKS
+- Provide community visibility on falco compatability through test results
+## Non-Goals
+- To replace the existing CICD pipeline
+
+
+## Proposal
+
+### Use Cases
+- Provide badges of test results to increase user confidence on multiple platform/versions.
+- Provide central UI for checking test logs/results during PR's
+- Provide distributed method for handling PR merging.
+- Utilize Github PR commands to control testing workflow such as re-tests
+
+
+### Prerequisites
+Install Prow on AWS EKS, the only prerequisite is that we have an AWS account, and an S3 bucket to store artifacts for Prow.
+
+### Design Detail
+
+Proposed install components
+- Plank
+- Hook
+- GhProxy
+- PodUtilities
+- Horologium
+- Tide (will look into difficulty)
+- Spyglass
+- Crier
+- Testgrid
+- Triage
+- Deck
+
+Open investigation items
+- Slack reporter to Falco K8s channel
```

Comment:
I'm okay with this if we report to another #falco-something channel, not the main one.